apps/admin/webpack.renderer.config.js

```diff
@@ -20,13 +20,18 @@ rules.push(
   },
 );
 
-const mocked = require('./webpack.mocked.config');
+// In development mode, we mock most external libraries
+const mockOptions = process.env.NODE_ENV !== 'production'
+  ? require('../../__mocks__/mock_webpack')
+  : {};
+
 const baseOptions = {
   module: {
```

Comment:
What is that file for?

Reply:
it's the single injection point.  If a webpack process merges that file, then all of the external services are automatically mocked